### PR TITLE
[literally gucci gang] RnD techweb rework

### PR DESCRIPTION
**Pull request by ma44**
@LetterN Note; will conflict for the subsystem research if this goes through, though it will be a easy value change

### CODE REVIEW

File: code/controllers/subsystem/research.dm [hunk 1]
```diff
@@ -24,8 +24,8 @@ SUBSYSTEM_DEF(research)
 	var/list/errored_datums = list()
 	var/list/point_types = list()				//typecache style type = TRUE list
 	//----------------------------------------------
-	var/list/single_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 54.3)
-	var/multiserver_calculation = FALSE
+	var/list/single_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 250)
+	var/multiserver_calculation = TRUE
```
Comment (LetterN) [inline code/controllers/subsystem/research.dm]: >build multiple servers
>get max rnd within 30 min into the game because of >>250 points per second

Comment (LetterN) [reply code/controllers/subsystem/research.dm]: also the
`//Aiming for 1.5 hours to max R&D
 	//[88nodes * 5000points/node] / [1.5hr * 90min/hr * 60s/min]
 	//Around 450000 points max???
`
is not accurate, it's just an estimate

File: code/game/objects/items/RCD.dm [hunk 2]
```diff
@@ -646,6 +647,63 @@ RLD
...
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rcd"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	var/circuit = 0
+
+/obj/item/circuitboardthing/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/circuitboard))
+		if(!circuit)
+			user << "You insert the [W.name] into the [src]."
```
Comment (LetterN) [inline code/game/objects/items/RCD.dm]: to_chat(src, stuff)

File: code/game/objects/items/RCD.dm [hunk 3]
```diff
@@ -646,6 +647,63 @@ RLD
...
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rcd"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	var/obj/item/gun/energy/gun = null
+
+/obj/item/portaturretconstruct/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/gun/energy))
+		if(!gun)
+			user << "You insert the [W.name] into the [src]."
```
Comment (LetterN) [inline code/game/objects/items/RCD.dm]: to_chat(user, stuff

File: code/game/objects/items/RCD.dm [hunk 4]
```diff
@@ -646,6 +647,63 @@ RLD
...
+			src.gun = new W.type(src)
+			qdel(W)
+	if(istype(W, /obj/item/screwdriver))
+		if(gun)
+			user << "You remove the [gun.name] from the [src]."
```
Comment (LetterN) [inline code/game/objects/items/RCD.dm]: to_chat

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -835,53 +838,44 @@
...
+	display_name = "Advanced Tool Technology"
+	description = "Things used by the really advanced people."
 	prereq_ids = list("biotech","engineering")
-	boost_item_paths = list(/obj/item/gun/energy/alien, /obj/item/scalpel/alien, /obj/item/hemostat/alien, /obj/item/retractor/alien, /obj/item/circular_saw/alien,
-	/obj/item/cautery/alien, /obj/item/surgicaldrill/alien, /obj/item/screwdriver/abductor, /obj/item/wrench/abductor, /obj/item/crowbar/abductor, /obj/item/multitool/abductor,
-	/obj/item/weldingtool/abductor, /obj/item/wirecutters/abductor, /obj/item/circuitboard/machine/abductor, /obj/item/abductor_baton, /obj/item/abductor)
+	design_ids = list("industrcd", "circuitmaker", "turretmaker")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 20000
 	hidden = TRUE
```
[the remark this answers is not in the document]
Comment (ma44) [reply code/modules/research/techweb/all_nodes.dm]: >tfw forgot hidden = true

**Comment by AndrewL97** [comment]
Hey @LetterN and @ma44 ,
I know nothing about new TG rnd, so I can't assist much in reviewing this. Let me know if you need me for anything on it though, otherwise @LetterN its your discretion to merge this.

**Comment by LetterN** [comment]
i'l fix the conflicts

**Pull request merged**